src/docs/release/breaking-changes/scaffold-messenger.md

```diff
@@ -95,6 +96,22 @@ Typically, the ScaffoldMessenger widget is introduced by the MaterialApp
 at the top of your application widget tree.
 ```
 
+The `ScaffoldMessenger` will also assert if attempting to show `SnackBar`s to nested `Scaffold`s. If
+a nested set of `Scaffold`s were to share the same `ScaffoldMessenger`, then duplicate `SnackBar`s
+could appear in your UI. In order to control the scope of `SnackBar`s and which `Scaffold`s receive
+them, place a `ScaffoldMessenger` in between. When trying to present a `SnackBar` in this situation,
+the follow assertion error will be thrown:
+
+```
+Nested Scaffolds have registered with the ScaffoldMessenger.
+If nested Scaffolds were to share the same ScaffoldMessenger, then
+all would receive a SnackBar at the same time, resulting in multiple
+SnackBars in your UI.
+This is typically resolved by putting a ScaffoldMessenger in
+between the levels of nested Scaffolds. Doing so will set a separate
+SnackBar scope for these Scaffolds.'
+```
+
```

Comment:
How about:

```
If multiple SnackBars appear in your UI, you have likely
registered these Scaffolds with the same ScaffoldManager.
You can usually resolve this by putting a ScaffoldMessenger in
between the levels of nested Scaffolds so that each SnackBar
has its own scope.
```

Reply:
Hm, multiple SnackBars would not actually show up, this error would be thrown if they were going to in order to prevent it.

Reply:
Ah, OK! That wasn't clear to me. So, how about:

```
The `ScaffoldManager` throws an assertion error when attempting to show a `SnackBar` to nested `Scaffold`s, which would display duplicate `SnackBar`s if allowed. To fix this, insert  a`SnackBar` between each nested `Scaffold` to set a separate scope for each.
```

The `ScaffoldMessenger` will also assert if attempting to show `SnackBar`s to nested `Scaffold`s. If a nested set of `Scaffold`s were to share the same `ScaffoldMessenger`, then duplicate `SnackBar`s between the levels of nested Scaffolds. Doing so will set a separate SnackBar scope for these Scaffolds.'